src/screens/Feed.js

```diff
@@ -30,7 +30,7 @@ export default class FeedScreen extends React.Component {
     const isLiked = likers[uid] === true ? true : false; // if i liked it
     const isBookmarked = bookmarkers[uid] === true ? true : false;
     return (
-      <TouchableWithoutFeedback onPress={() => { }}>
+      <TouchableWithoutFeedback onPress={() => {}}>
```

Comment:
Property onPress is not required, and thus we can remove it (If unless it is needed for something else down the road -- Which then I recommend writing comments. )
https://reactnative.dev/docs/touchablewithoutfeedback

Reply:
Remove <TouchableWithoutFeedback onPress={() => { }}>